docs/sphinx/examples/1d_stencil.rst

```diff
@@ -150,12 +150,12 @@ called unless the data from the neighboring partitions have arrived. Therefore
 it should come as no surprise that we synchronize this operation with another
 instance of dataflow (found in ``heat_part``). This dataflow is passed futures
 to the data in the current and surrounding partitions by calling ``get_data()``
-on each respective partition. When these futures are ready dataflow passes then
-to the ``unwrapped`` function, which extracts the shared_array of doubles and
+on each respective partition. When these futures are ready dataflow passes them
+to the ``unwrapping`` function, which extracts the shared_array of doubles and
```

Comment:
Same here, the HPX facility is called `unwrapped`.

Reply:
I renamed `unwrapped` back to `unwrapping`.